### PR TITLE
Incorporate updated slides into base enterprise branch

### PR DESCRIPTION
**Pull request by derekpierre**
Updated some enterprise slides related to the slides created for Target.

### CODE REVIEW

File: slides.tex [hunk 1]
```diff
@@ -117,6 +127,14 @@
         \end{itemize}
     \end{frame}
 
+    \begin{frame}
+        \frametitle{Blockchain Smart Contract Agnostic}
```
Comment (mswilkison) [inline slides.tex]: Blockchain & Smart Contract Platform Agnostic

File: slides.tex [hunk 2]
```diff
@@ -153,10 +171,19 @@
         \end{figure}
     \end{frame}
 
+    \begin{frame}
```
Comment (mswilkison) [inline slides.tex:170]: Move this after data lake use case slide

Comment (derekpierre) [reply slides.tex:170]: Slides updated.

**Review by michwill — COMMENTED** [review]
* "Not scalable" -> I think "not scalable unless decrypted server-side"
* "NuCypher made investments in this area" - confusing. I think, "NuCypher invested efforts in this area". Otherwise, people may think of money investments in some companies

**Comment by derekpierre** [comment]
@michwill re: "not scalable" - Even if decrypted server-side, unless all of the recipients are known at sharing time, it isn't scalable anyway, no? Unless you mean the data is stored in a decrypted form on the server...

**Comment by derekpierre** [comment]
@michwill, @mswilkison any further concerns with the pull request?

**Comment by derekpierre** [comment]
Incorporated the fix for the text rendering issues that @michwill observed in this PR.

**Comment by mswilkison** [comment]
Let's swap the logo on the first slide with the version that also has our
wordmark (like on the website homepage) and delete the 'NuCypher' text.

And once we do that it might make sense to move the logo above the the
presenter name/date/event depending on how it looks.

**Comment by derekpierre** [comment]
Logo updated. I think it looks great!

Note that the logo in the "More Information" slide also got updated.

**Review by mswilkison — APPROVED** [review]

**Comment by mswilkison** [comment]
Nice! Makes sense to push that same change to `master` imo

**Comment by derekpierre** [comment]
I plan to, and for my "target" branch as well.

But I may end up doing it manually because the merging would be a nightmare.

**Pull request merged**